mobile/src/lib/cozy-helper.js

```diff
@@ -82,6 +82,27 @@ export function refreshFolder (dispatch, getState) {
   }
 }
 
+export const startRepeatedReplication = (dispatch, getState) => {
+  const options = {
+    onError: onError(dispatch, getState),
+    onComplete: refreshFolder(dispatch, getState)
+  }
+  cozy.client.offline.startRepeatedReplication('io.cozy.files', 15, options)
+}
+
+export const startFirstReplication = (dispatch, getState) => {
+  const options = {
+    onError: onError(dispatch, getState),
+    onComplete: () => {
+      dispatch(setFirstReplication(true))
+      startRepeatedReplication(dispatch, getState)
+    }
+  }
+  cozy.client.offline.replicateFromCozy('io.cozy.files', options).then(() => {
+    console.log('End of Replication')
```

Comment:
oups I don't remove this log after my refactoring. But It highlights a bug [I specified on Trello](https://trello.com/c/DDUppsPT/222-requiresetup-est-appele-plusieurs-fois).